src/sentry/static/sentry/app/components/suggestProjectCTA.tsx

```diff
@@ -124,12 +126,37 @@ class SuggestProjectCTA extends React.Component<Props, State> {
         userAgentMatches: !!matchedUserAgentString,
         hasMobileProject: this.hasMobileProject,
         snoozedOrDismissed: isDismissed,
+        mobileEventBrowserName: mobileEventResult?.browserName || '',
+        mobileEventClientOsName: mobileEventResult?.clientOsName || '',
       },
       this.props.organization,
       {startSession: true}
     );
   }
 
+  async checkOrgHasMobileEvent(): Promise<MobileEventResult> {
+    const {api, organization} = this.props;
+    return await api.requestPromise(
```

Comment:
You don't need to make this function async or add the await here, it already returns a promise